tensorrt_llm/executor/worker.py

```
@@ -406,6 +406,10 @@ def _enqueue_request(self, request: GenerationRequest) -> int:
         context_phase_params = None
         request_type = tllm.RequestType.REQUEST_TYPE_CONTEXT_AND_GENERATION
         if request.disaggregated_params is not None:
+            assert (
+                not self._is_pytorch_backend
+                or self.engine.kv_cache_transceiver is not None
+            ), "kv_cache_transceiver is disabled, please set 'cache_transceiver_config: backend: ` in config file for disaggregated serving"
```

Ruff (0.12.2)

Line too long (140 > 120)

(E501)